test/oban/job_test.exs

```diff
@@ -11,10 +12,20 @@ defmodule Oban.JobTest do
       assert changeset.changes[:scheduled_at]
     end
 
+    property "scheduling a job for the future using a tuple" do
+      check all schedule_in <- tuple({integer(), time_unit()}) do
+        changeset = Job.new(%{}, worker: Fake, schedule_in: schedule_in)
+        assert changeset.changes[:state] == "scheduled"
+        assert changeset.changes[:scheduled_at]
```

Comment:
Love the property test here! It would be nice to limit the value to positive integers and then make an assertion that `scheduled_at` was both a DateTime and in the future, rather than a plain truthy check.